docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Exciting MUI X improvements: v6.11.0
+description: Support for time zones, Charts in alpha, Data Grid filtering, and more.
+date: 2023-08-07T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we've continuously rolled out new major features.
+Now that we've reached [MUI X v6.11.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Date Pickers](#date-pickers)
+  - [Support for time zones](#support-for-time-zones)
+  - [Digital clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and paste](#copy-and-paste)
+  - [Filter on column headers](#filter-on-column-headers)
+  - [Performance enhancements: smoother scrolling and expanding rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - alpha version 🎁](#charts-alpha-version)
+- [What's next](#whats-next)
+- [Feedback](#feedback)
+
+## Date Pickers
+
+### Support for time zones
+
+No more hassle displaying dates in the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about conversion logic or maintaining consistency with your backend.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
```

Comment:
Do we need the video controls on this loop video? If we follow https://www.notion.so/mui-org/Tips-for-high-quality-media-5a1686758f4f4267989b67a1c2c270da?pvs=4#5f9e23d043a741dcbc1304ac9e9c1ba3 we should do:

```suggestion
<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop>
```

and in the one below.

Reply:
I like adding the control attribute just in case the reader finds the loop annoying when reading through the blog. Imagine reading through a blog and theres a colorful video playing nonstop. IMO, I would find it distracting. 

Will commit your suggestion for now and pose the question in notion. We can discuss there